HVZ/HVZ/api/views.py

```diff
@@ -34,12 +35,18 @@ def json_get_all_emails(request):
 class Mailer(FormView):
     form_class = MailerForm
     template_name = "api/mailer.html"
+
+    # return url
     success_url = '/success/'
 
     def dispatch(self, *args, **kwargs):
+        # used to display the form
         return super(Mailer, self).dispatch(*args, **kwargs)
 
     def get_success_url(self):
+        # after the mail is sent successfully, it goes to the success page
+        # At this point, it is the same as registration success page
+        # in the future, we will have more details 
```

Comment:
It's good that we recognized this! As for a solution, what page should we redirect to upon successfully sending an email? Should we simply return to the blank mailer form, or should we create a new success page?